ui/app/controllers/vault/cluster/unseal.js

```diff
@@ -6,10 +6,12 @@ export default Controller.extend({
 
   actions: {
     transitionToCluster(resp) {
-      return this.get('model').reload().then(() => {
-        this.get('wizard').transitionTutorialMachine(this.get('wizard.currentState'), 'CONTINUE', resp);
-        return this.transitionToRoute('vault.cluster', this.get('model.name'));
-      });
+      return this.get('model')
+        .reload()
+        .then(() => {
+          this.get('wizard').transitionTutorialMachine(this.get('wizard.currentState'), 'CONTINUE', resp);
+          return this.transitionToRoute('vault.cluster', this.get('model.name'));
+        });
```

Comment:
seems like the prettier update may have changed some things - that or your editor config is different than the prettier settings we're using - not sure which, but we can sort that later :)